torch/_torch_docs.py

```diff
@@ -1130,6 +1130,26 @@ def merge_dicts(*dicts):
     tensor([ True, False, False])
 """.format(**common_args))
 
+add_docstr(torch.broadcast_to,
+           r"""
+broadcast_to(input, shape) -> Tensor
+
+Broadcasts :attr:`input` to the :attr:`shape`.
```

Comment:
"to the :attr:\`shape\`" -> to the shape :attr:\`\shape\`"

Sorry this didn't format correctly the first time I suggested it. I think the formatting is fixed now.

Reply:
Thanks so much for the kind suggestions for improving the doc, it has been updated accordingly.